Makefile

```diff
@@ -179,11 +179,11 @@ ifndef BUNDLE_PATH
 export BUNDLE_PATH = $(HOME)/Downloads/crc_libvirt_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
 endif
 integration:
-	@go test -timeout=60m $(REPOPATH)/test/integration -v
+	@go test -timeout=60m $(REPOPATH)/test/integration -v -ginkgo.noColor
```

Comment:
@cfergeau if you are talking about centos CI run then to get the colored output you always need to click full logs and you wlll have nice colored logs, this is really helped me to just go to error message quickly.

Reply:
The other similar change is about CentOS CI, this one is about prow but is not working as expected.
They were mostly added as experiment/to help debug this PR, I can move them away for now. But default output of CentOS CI is not readable, clicking on "unformatted text" is also unreadable, one has to know there is a 3rd non-default link which will give readable output. Suboptimal :(